### DISCUSSION TITLE
PSA > 45

### DISCUSSION THREAD

**Discussion by sstreitmatter** [ncidose/ncidosetools#32] [NCIRF]
I am trying to calculate dose for a shoulder axillary view, but am limited to +/- 45 degree on the PSA angle.  I am using the "rotated" phantom, but to get to a representative axillary view I need a PSA of 90.  I can override this in batch mode, but it won't run the simulation.  Is there a reason it is limited to +/- 45?  Can it be increased to +/- 90?  Thanks.
<img width="1206" alt="Screenshot 2024-08-09 at 12 59 15 PM" src="https://github.com/user-attachments/assets/3c4b99da-73f4-4330-9c94-e4e4800714b7">

**Comment by choonsiklee** [comment]
We will include this feature into the next official release in October.

**Reply by sstreitmatter** [reply]
Any update on this feature?  Thanks!

-Seth

**Reply by ncidoseteam** [reply]
We’re currently working on implementing this feature, but it requires additional consideration of the source-to-isocenter distance (SID) to ensure the source remains within the phantom anatomy. Could you clarify how your machine can accommodate this angle of irradiation?

**Reply by sstreitmatter** [reply]
Here are some examples of general radiographic views that would need the additional range of PSA angle:
<img width="917" alt="Screenshot 2025-05-19 at 11 13 48 AM" src="https://github.com/user-attachments/assets/f7eaa021-a6a7-49e3-98ef-643c4a4903d2" />
<img width="908" alt="Screenshot 2025-05-19 at 11 14 18 AM" src="https://github.com/user-attachments/assets/593c667d-b0b1-4123-befa-722c954b91d1" />
<img width="908" alt="Screenshot 2025-05-19 at 11 14 51 AM" src="https://github.com/user-attachments/assets/33d834eb-7707-490e-94f9-b0b28e159f27" />

**Reply by ncidoseteam** [reply]
Thank you for sharing these diagrams. We will incorporate this feature into the upcoming bi-annual upgrade scheduled for June 2025.
